qa/tasks/cephfs/test_volumes.py

```diff
@@ -136,6 +136,77 @@ def test_subvolume_create_and_rm(self):
         # verify trash dir is clean
         self._wait_for_trash_empty()
 
+    def test_subvolume_resize(self):
+        # tests the 'fs subvolume resize' command
+
+        # create subvolume
+        subvolname = self._generate_random_subvolume_name()
+        osize = self.DEFAULT_FILE_SIZE*1024*1024
+        self._fs_cmd("subvolume", "create", self.volname, subvolname, "--size", str(osize))
+
+        # make sure it exists
+        subvolpath = self._get_subvolume_path(self.volname, subvolname)
+        self.assertNotEqual(subvolpath, None)
+
+        # resize the subvolume with double size of the original size
+        nsize = osize*2
+        self._fs_cmd("subvolume", "resize", self.volname, subvolname, str(nsize))
```

Comment:
Always best to break these into specific tests.

* Test that the subvolume can grow; verify the quota on the directory rstat.
* Test that the subvolume can shrink; verify the quota on the directory rstat.
* Test that the subvolume quota can shrink below its current use; verify quota/usage on dir rstat.
* Test that the subvolume quota can be expanded on a full subvolume and future writes succeed.
* What does resize=0 do? Write a test for it.